components/Input/NumericInput.js

```diff
@@ -37,6 +39,16 @@ class NumericInput extends React.PureComponent {
 		});
 	}
 
+	blur(e) {
+		this.end();
+	}
+
+	end() {
+		if (this.props.onEnd) {
```

Comment:
Instead of checking whether `onEnd` exists, put a default `noop` in the component's `defaultProps`, like for `onChange`.

Reply:
And let's pass the `e` event from blur to `end` as well.